### PR TITLE
feat(plasma): Online Plasma Input Fetcher

### PR DESCRIPTION
**Pull request by refcell**
**Description**

Stubs out an online plasma input fetcher.

### CODE REVIEW

**Comment by refcell** [comment]
* **#167** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/kona/167?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @refcell and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by refcell** [comment]
Ah so this is where the ci was breaking from the now-closed large pr @clabby

**Comment by refcell** [comment]
It's very likely due to a bad feature flag being enabled.

File: crates/plasma/src/lib.rs [hunk 1]
```diff
@@ -11,5 +11,10 @@ pub mod source;
 pub mod traits;
 pub mod types;
 
+#[cfg(feature = "online")]
+pub mod online;
+#[cfg(feature = "online")]
+pub use online::OnlinePlasmaInputFetcher;
```
Comment (clabby) [inline crates/plasma/src/lib.rs:17]:
```suggestion
#[cfg(feature = "online")]
mod online;
#[cfg(feature = "online")]
pub use online::OnlinePlasmaInputFetcher;
```

**Review by clabby — APPROVED** [review]

**Pull request merged**